game-server/data/handlers/ai/instance/empyreanCrucible/VanktristAI.java

```diff
@@ -0,0 +1,112 @@
+package ai.instance.empyreanCrucible;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.Future;
+
+import com.aionemu.commons.utils.Rnd;
+import com.aionemu.gameserver.ai.AIName;
+import com.aionemu.gameserver.ai.HpPhases;
+import com.aionemu.gameserver.model.gameobjects.Creature;
+import com.aionemu.gameserver.model.gameobjects.Npc;
+import com.aionemu.gameserver.model.gameobjects.player.Player;
+import com.aionemu.gameserver.skillengine.model.SkillTemplate;
+import com.aionemu.gameserver.utils.PacketSendUtility;
+import com.aionemu.gameserver.utils.ThreadPoolManager;
+import com.aionemu.gameserver.world.geo.GeoService;
+
+import ai.AggressiveNoLootNpcAI;
+
+/**
+ * @author w4terbomb
+ */
+@AIName("vanktrist")
+public class VanktristAI extends AggressiveNoLootNpcAI implements HpPhases.PhaseHandler {
+
+	private final HpPhases hpPhases = new HpPhases(100, 75, 50);
+
+	private Future<?> phaseTask;
+
+	private VanktristAI(Npc owner) {
+		super(owner);
+	}
+
+	@Override
+	protected void handleAttack(Creature creature) {
+		super.handleAttack(creature);
+		hpPhases.tryEnterNextPhase(this);
+	}
+
+	@Override
+	public void handleHpPhase(int phaseHpPercent) {
+		switch (phaseHpPercent) {
+			case 100 -> PacketSendUtility.broadcastMessage(getOwner(), 0); // FIXME
+			case 75 -> {
+				PacketSendUtility.broadcastMessage(getOwner(), 1500210); // STR_CHAT_IDArena_STAGE7_D_R1_Die TODO correct msg
+				startPhaseTask();
+			}
+			case 50 -> PacketSendUtility.broadcastMessage(getOwner(), 1500211); // STR_CHAT_IDArena_STAGE7_L_R2_Skill1 TODO different msg for L/D
```

Comment:
The message IDs are not correct.
Example: `STR_CHAT_IDArena_STAGE7_D_R1_Die`
- `D` means dark faction (Asmodians). Each faction has their own message (Elyos have `L`)
- `R1` means round 1, `R2` round 2, and so on. Each round has their own messages.
- `Die` means the message belongs to the death event of the npc.
There is also `Skill1`, `Skill2` and so on, which corresponds to a specific skill cast event. Not sure if messages are tied to certain skill IDs or if it's a random chance on each skill cast.